charts/cluster-autoscaler/values.yaml

```diff
@@ -346,9 +346,11 @@ service:
   annotations: {}
   # service.labels -- Labels to add to service
   labels: {}
-  # service.externalIPs -- List of IP addresses at which the service is available. Ref: https://kubernetes.io/docs/user-guide/services/#external-ips.
+  # service.externalIPs -- List of IP addresses at which the service is available. Ref: https://kubernetes.io/docs/concepts/services-networking/service/#external-ips.
   externalIPs: []
 
+  # service.clusterIP -- IP address to assign to service
+  clusterIP: ""
```

Comment:
I know the original issue noted this was missing, but is there a reason you would actually set clusterIP?   I would think that we should actually remove it from the template itself?